docs/editor/vscode-web.md

```diff
@@ -95,13 +105,23 @@ In some cases, you will want to access a different environment that has the abil
 
 The GitHub Repositories extension makes it easy for you to clone the repository locally, reopen it on the desktop, or create a GitHub codespace for the current repository (if you have the [GitHub Codespaces](https://marketplace.visualstudio.com/items?itemName=GitHub.codespaces) extension installed and access to create GitHub codespaces). To do this, use the **Continue Working On...** command available from the Command Palette (`kbstyle(F1)`) or click on the Remote indicator in the Status bar.
 
-## Use your own compute power with the VS Code Server
+## Saving and sharing work
 
-You may develop against another machine in VS Code for the Web using the VS Code Server.
+When working on a local file in the web, your work is saved automatically if you have [Auto Save](/docs/editor/codebasics.md#save-auto-save) enabled. You can also save manually as you do when working in desktop VS Code (for example **File** > **Save**).
 
-The VS Code Server is a private preview service you can run on a remote development machine, like your desktop PC or a virtual machine (VM). It allows you to securely connect to that remote machine from anywhere through a vscode.dev URL, without the requirement of SSH. This lets you "bring your own compute" to vscode.dev, enabling additional scenarios like running your code in the browser.
+When working on a remote repository, your work is saved in the browser's local storage until you commit it. If you open a repo or pull request using GitHub Repositories, you can push your changes in the Source Control view to persist any new work.
 
-You may learn more about the VS Code Server in its [documentation](https://aka.ms/vscode-server-doc).
+You can also continue working in other environments via [**Continue Working On**](#continue-working-in-a-different-environment).
+
+The first time that you use **Continue Working On** with uncommitted changes, you will have the option to bring your edits to your selected development environment using **Cloud Changes**, which uses a VS Code service to store your pending changes. This is described further in the [GitHub Repositories](/docs/sourcecontrol/github.md#continue-working-on) doc.
+
+## Use your own compute with Remote - Tunnels
```

Comment:
Can you also change this to just "with Remote Tunnels" and also update the header link above?
Sometimes we have trouble with linking to dashes in headers and I think calthe feature is "Remote Tunnels" and it's only the extension that is "Remote - Tunnels"

Reply:
Updated line 118 and the earlier link to it, let me know if there were other spots!